### ISSUE TITLE
Get final version of Form ONRR-4054 Frequently Asked Questions

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1736]
Cody, this was the new one that wasn't finalized.

**Comment by mcharg** [comment]
Sent Christen Watkins an email asking when she might have the final FAQ.

**Comment by mcharg** [comment]
The document owner said we can post a Word doc instead of a PDF.

**Comment by jkwonatx** [comment]
[https://app.zenhub.com/files/210892300/345cfe59-554e-49ea-b978-536e1798a6aa/download](https://app.zenhub.com/files/210892300/345cfe59-554e-49ea-b978-536e1798a6aa/download)

To fix the accessibility flag for no heading, I changed the style of the title on the first page from Normal to Heading ONRR (which I created by modifying Heading 1). The accessibility checker on Word should now show "good to go"

**Comment by mcharg** [comment]
@jkwonatx Looks good to me. Can you reply to Christen's email?

**Comment by jkwonatx** [comment]
@mcharg I'll reply to the email as well once I complete the manual check.

**Comment by jkwonatx** [comment]
@mcharg I've completed the manual check. I used the checklist found on the ONRResources Sharepoint folder. 

I've made two more changes in addition to the above: 1. replaced the existing table of contents with a properly formatted one 2. deleted the numbers in front of FAQ titles as I don't think it adds to the message and it was formatted wrong, and fixing it was going to complicate the bullets and numbering. 

I'll email by replying to all the final version along with the manual checklist.
